src/core/public/apm_system.ts

```diff
@@ -87,6 +74,45 @@ export class ApmSystem {
     });
   }
 
+  /* Hold the page load transaction open, until all resources actually finish loading */
+  private holdPageLoadTransaction(apm: ApmBase) {
+    this.pageLoadTransaction = apm.getCurrentTransaction();
+
+    // Keep the page load transaction open until all resources finished loading
+    if (this.pageLoadTransaction && this.pageLoadTransaction.type === 'page-load') {
+      // @ts-expect-error 2339
```

Comment:
`2239` is a bit cryptic
```suggestion
      // @ts-expect-error 2339 block is a private property of Transaction interface
```